### DISCUSSION TITLE
fonctionnalité d'import des fichiers CSV fournis par ENEDIS

### DISCUSSION THREAD

**Discussion by alainvtr** [MyElectricalData/myelectricaldata_import#73] [Ideas]
Bonjour,

Vu qu'il est possible de récupérer pas mal de données sur la conso via le site d'ENEDIS (1 zip avec 4 CSV à priori), serait-il possible de prévoir une fonction d'import via une petite IHM qui serait mise à dispo ?
Cela permettrait peut-être d'accélérer le processus de récupération et soulagerait aussi la gateway ?

Cordialement.

**Comment by cvalentin-dkt** [comment]
C'est une possibilité.
L'idée à déjà était soumise dans [ici](https://github.com/m4dm4rtig4n/enedisgateway2mqtt/issues/68)
